doc/en-us/dev/java/entity-ai/behavior_group.md

```diff
@@ -1,39 +1,39 @@
-# 第六章 行为组 - 组件的集合
+# Chapter 6 Behavior Groups - Collections of Components
 
 _**author: daoge_cmd**_
 
-**Waiting for translation, if you are interested in translation, welcome to contribute.**  
+**The translation made by AzaleeX contributed to the PowerNukkitX documentation**  
 
-## 1.0 所有组件的容器
+## 1.0 Containers for all components
 
-事实上，行为组```BehaviorGroup```并不只是行为的容器，其实际上包含了一类生物的全部逻辑组件，本章介绍的行为只是其组件的一部分。一个完整的行为组包含：
+In fact, the behavior group ```BehaviorGroup``` is not just a container for behaviors, but actually contains all the logical components of a class of creatures, and the behaviors presented in this chapter are only some of its components. A complete behavior group contains:
 
-- 行为，核心行为
-- 记忆存储
-- 传感器
-- 寻路器
-- 控制器
+- Behavior, Core Behavior
+- Memory Storage
+- Sensors
```

Comment:
It would be better to unify the singular and plural forms here

Reply:
I don't see what I need to change in all of this.

Reply:
Maybe remove the `s` in Sensors?

Reply:
fix on this commit d476ae851733db3d3eca74a962c507167620b074